Formula/sdl12-compat.rb

```diff
@@ -0,0 +1,26 @@
+class Sdl12Compat < Formula
+  desc "SDL 1.2 compatibility layer that uses SDL 2.0 behind the scenes"
+  homepage "https://github.com/libsdl-org/sdl12-compat"
+  url "https://github.com/libsdl-org/sdl12-compat/archive/refs/tags/release-1.2.52.tar.gz"
+  sha256 "5bd7942703575554670a8767ae030f7921a0ac3c5e2fd173a537b7c7a8599014"
+  license all_of: ["Zlib", "MIT-0"]
+  head "https://github.com/libsdl-org/sdl12-compat.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "sdl2"
+
+  conflicts_with "sdl"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args,
+           "-DSDL2_PATH=#{Formula["sdl2"].prefix}",
+           "-DSDL12DEVEL=0", "-DSDL12TESTS=1"
```

Comment:
Can you elaborate why you chose these.

Why do we need any of the test programs (`SDL12TESTS`)?
Why should we not include SDL headers (`SDL12DEVEL`), particularly given this conflicts with sdl1?

Reply:
> Can you elaborate why you chose these?
> 
> Why do we need any of the test programs (`SDL12TESTS`)? Why should we not include SDL headers (`SDL12DEVEL`), particularly given this conflicts with sdl1?

Both are on by default; I don't recall why I opted for tests but I know why I turned off "devel": I didn't know that's what it was.  I assumed it was something only for maintainers to have on.  I'll flip them both.

```suggestion
           "-DSDL12DEVEL=1", "-DSDL12TESTS=0"
```